lib/services/rocket.py

```diff
@@ -63,9 +63,12 @@ def from_rocket_model(cls, rocket: Rocket) -> Self:
         rocketpy_rocket.evaluate_static_margin()
 
         # FinSet
-        finset = cls.get_rocketpy_finset(rocket.fins, rocket.fins.fins_kind)
-        rocketpy_rocket.aerodynamic_surfaces.add(finset, finset.position)
-        rocketpy_rocket.evaluate_static_margin()
+        rocketpy_finset_list = cls.get_rocketpy_finset_list_from_fins_list(
+            rocket.fins
+        )
+        for finset in rocketpy_finset_list:
+            rocketpy_rocket.aerodynamic_surfaces.add(finset, finset.position)
+            rocketpy_rocket.evaluate_static_margin()
```

Comment:
The `evaluate_static_margin()` is really time-consuming, you only need to call it once, after all the fins have already been set.

Reply:
patched in [95aaf32](https://github.com/RocketPy-Team/Infinity-API/pull/33/commits/95aaf329a65646f9940480bee6b370ef71bc3059)